packages/react/ds-app-launchpad/package.json

```diff
@@ -46,14 +46,8 @@
     "@canonical/biome-config": "^0.9.0-experimental.1",
     "@canonical/typescript-config-react": "^0.9.0-experimental.1",
     "@chromatic-com/storybook": "^3.2.2",
-    "@storybook/addon-essentials": "^8.4.7",
-    "@storybook/addon-interactions": "^8.4.7",
-    "@storybook/addon-links": "^8.4.7",
-    "@storybook/addon-onboarding": "^8.4.7",
-    "@storybook/addon-themes": "^8.4.7",
     "@storybook/blocks": "^8.4.7",
     "@storybook/react": "^8.4.7",
-    "@storybook/react-vite": "^8.4.7",
```

Comment:
REALLY nice to see these being removed and abstracted into our config package. This is demonstrating the value of this package.